verilog/analysis/checkers/parameter_name_style_rule_test.cc

```diff
@@ -37,11 +37,9 @@ TEST(ParameterNameStyleRuleTest, AcceptTests) {
       {"module foo; endmodule"},
       {"module foo (input bar); endmodule"},
       {"module foo; localparam Bar = 1; endmodule"},
-      {"module foo; localparam type Bar_1 = 1; endmodule"},
       {"module foo; localparam int Bar = 1; endmodule"},
       {"module foo; parameter int HelloWorld = 1; endmodule"},
       {"module foo #(parameter int HelloWorld_1 = 1); endmodule"},
-      {"module foo #(parameter type Foo); endmodule"},
```

Comment:
Can we keep these in `AcceptTests` so that we continue to verify that they trigger no violation?

Reply:
Sure, that makes sense!